cvat-ui/src/components/annotation-page/standard-workspace/controls-side-bar/controls-side-bar.tsx

```diff
@@ -114,6 +116,8 @@ export default function ControlsSideBarComponent(props: Props): JSX.Element {
                 ActiveControl.OPENCV_TOOLS,
             ].includes(activeControl);
 
+            if (!labels.length) return;
```

Comment:
Instead I would suggest to set handlers and shortcuts to empty dictionary if no labels.